src/builder/builder.go

```diff
@@ -498,8 +498,15 @@ func (b *Builder) AddRPMList(rpms []os.FileInfo) {
 			fmt.Println("ERROR: RPM is not valid! Please make sure it was built correctly.")
 			os.Exit(1)
 		} else {
-			fmt.Printf("Copying %s\n", rpm.Name())
-			helpers.CopyFile(b.Repodir+"/"+rpm.Name(), b.Rpmdir+"/"+rpm.Name())
+			if _, err = os.Stat(b.Repodir + "/" + rpm.Name()); err == nil {
+				continue
+			}
```

Comment:
Well, stat(2) can return many different errors. The ones that are most interesting (aside from ENOENT) are related to path traversal failures: EACCES, ELOOP, ENAMETOOLONG, and ENOTDIR. You can probably rule out EACCES, because mixer runs as root. The others are possible, though unlikely.